packages/shared/src/components/Sidebar.tsx

```diff
@@ -193,6 +221,10 @@ export default function Sidebar(): ReactElement {
           onRequestClose={() => setShowSettings(false)}
         />
       )}
+      <FeedFilters
```

Comment:
This can be loaded dynamically I think to reduce initial bundle size

Reply:
That makes sense though I will have to add another mechanism to display the animation.